app/views/record/_record.html.erb

```diff
@@ -1,59 +1,183 @@
-
-<%= field_table(@record, 'contributors', %w[value kind affiliation identifier mit_affiliated]) %>
-
-<%= field_table(@record, 'dates', %w[kind value range note]) %>
-
-<%= field_string(@record, 'edition') %>
-
-<%= field_list(@record, 'fileFormats') %>
-
-<%= field_string(@record, 'format') %>
-
-<%= field_table(@record, 'fundingInformation', %w[award_number award_uri funder_identifier funder_identifier_type funder_name]) %>
-
-<%= field_table(@record, 'holdings', %w[call_number collection format location note]) %>
-
-<%= field_object(@record, 'identifiers') %>
-
-<%= field_list(@record, 'languages') %>
-
-<%= field_table(@record, 'links', %w[kind text url restrictions]) %>
-
-<%= field_string(@record, 'literaryForm') %>
-
-<%= field_table(@record, 'locations', %w[kind value geopoint]) %>
-
-<%= field_object(@record, 'notes') %>
-
-<%= field_string(@record, 'numbering') %>
-
-<%= field_string(@record, 'physicalDescription') %>
-
-<%= field_list(@record, 'publicationFrequency') %>
-
-<%= field_list(@record, 'publicationInformation') %>
-
-<%= field_table(@record, 'relatedItems', %w[description item_type relationship uri]) %>
-
-<%= field_table(@record, 'rights', %w[description kind uri]) %>
-
-<%= field_string(@record, 'source') %>
-
-<%= field_string(@record, 'sourceLink') %>
-
-<%= field_object(@record, 'subjects') %>
-
-<%= field_list(@record, 'summary') %>
-
-<%= field_string(@record, 'timdexRecordId') %>
+<div class="gridband layout-3q1q wrap-full-record">
+  <div class="col3q box-content region full-record" data-region="Full record">
+
+    <h2 class="record-title">
+      <span class="sr">Title: </span>
+      <% if @record['title'].present? %>
+        <%= @record['title'] %>
+      <% else %>
+        No title provided for this item.
+      <% end %>
+    </h2>
+
+    <p>
+      <span class="record-type">
+        <% # we need to create separate displays for different types of content types, see https://github.com/MITLibraries/bento/pull/908/files#diff-6718f9b88f6f93e59d8c2509512dc40a62088430ff3cce1165ab414565949415L22-L57 for starting point but we should anticipate wanting to expand to many more unique content types %>
+        <span class="sr">Type</span> <%= @record['contentType'].join %></span>
+
+        <% if @record['citation'].present? %>
+
+          <span class="record-citation">
+            <%= @record['citation'] %>
+          </span>
+
+        <% else %>
+
+          <% if @record['dates'].map{|date| date if date['kind'] == 'Publication date'}.compact.present? %>
+            <span class="record-year">
+              : Published <%= date_parse(publication_date(@record)) %>
+            </span>
+          <% end %>
+          <% if @record['edition'].present? %>
+            <span class="record-edition">
+              : Edition <%= @record['edition'] %>
+            </span>
+          <% end %>
+        <% end %>
+    </p>
+
+    <% # we should helper this up to handle insanely large lists of authors %>
```

Comment:
Non-blocking: I'm happy to leave this until we implement the helper, but we should also deal with either de-duplicating contributors with multiple roles, or displaying those multiple roles in some way. At the moment we're just showing the same values multiple times.

That will also be an opportunity to decide if we want these names to be links (as they are on the results page) or plain text (as they are here now).

Reply:
I agree all around on this, but am not going to make changes at this time.